modules/@apostrophecms/schema/ui/apos/mixins/AposInputMixin.js

```diff
@@ -33,6 +34,9 @@ export default {
   },
   mounted () {
     this.$el.addEventListener('focusout', this.validateAndEmit);
+    if (this.triggerValidation) {
```

Comment:
We would want to `watch` this instead, right?

Reply:
Yes, and no need for a `key` in this case 👍